src/cache_cf.cc

```diff
@@ -2976,8 +2976,7 @@ dump_time_t(StoreEntry * entry, const char *name, time_t var)
 void
 parse_time_t(time_t * var)
```

Comment:
Please fix this API as well.

Reply:
This is out of scope for this PR - no callers of this API suffer from initialization TOCTOU issues. I will do a followup refactor PR instead.